urdf2webots/writeRobot.py

```diff
@@ -260,10 +268,10 @@ def URDFBoundingObject(robotFile, link, level, boxCollision):
                                                                                 boundingObject.geometry.box.z))
             robotFile.write(boundingLevel * indent + '}\n')
 
-        elif boundingObject.geometry.cylinder.radius != 0 and boundingObject.geometry.cylinder.length != 0:
+        elif boundingObject.geometry.cylinder.radius != 0 and boundingObject.geometry.cylinder.height != 0:
             robotFile.write(initialIndent + 'Cylinder {\n')
             robotFile.write((boundingLevel + 1) * indent + 'radius ' + str(boundingObject.geometry.cylinder.radius) + '\n')
-            robotFile.write((boundingLevel + 1) * indent + 'height ' + str(boundingObject.geometry.cylinder.length) + '\n')
+            robotFile.write((boundingLevel + 1) * indent + 'height ' + str(boundingObject.geometry.cylinder.height) + '\n')
```

Comment:
Also here we shouldn't write the radius if it is the default value. Same for the height.